### ISSUE TITLE
WARN the custom formats do not exist in the guide and will be skipped

### ISSUE THREAD

**Issue by 2fst4u** [recyclarr/recyclarr#110]
After the recent changes to Radarr schema not supporting "name" for the selection and moving to "trash_ids", I have changed my yaml over to `trash_ids` and added all the relevant trash IDs, commenting out the names. But now my log shows the above warning and quotes all my trash IDs that I'm trying to sync.

Is there something else that needs to be done? Here is an example of my config:

```
radarr:
  # Set the URL/API Key to your actual instance
  - base_url: http://radarr-service.radarr:80
    api_key: 

    # Which quality definition in the guide to sync to Radarr. Only choice right now is 'movie'
    quality_definition:
     type: movie
     preferred_ratio: 0.5

    # Set to 'true' to automatically remove custom formats from Radarr when they are removed from
    # the guide or your configuration. This will NEVER delete custom formats you manually created!
    delete_old_custom_formats: true

    custom_formats:
      # A list of custom formats to sync to Radarr. Must match the "name" in the importable JSON.
      # Do NOT use the heading names here, those do not work! These are case-insensitive.
      - trash_ids:
          - ed38b889b31be83fda192888e2286d83 # BR-DISK
          - 90cedc1fea7ea5d11298bebd3d1d3223 # EVO (no WEBDL)
          - 90a6f9a284dff5103f6346090e6280c8 # LQ
          - b8cd450cbfa689c0259a01d9e29ba3d6 # 3D
          - 1af239278386be2919e1bcee0bde047e # DD+ ATMOS
          - 1c7d7b04b15cc53ea61204bebbcc1ee2 # HQ
          - 403f3f6266b90439cacc1e07cae4dc2d # HQ-Remux
          - 26fa26253af4001701fedb56cec376dc # HQ-WEBDL
          - a570d4a0e56a2874b64e5bfa55202a1b # FLAC
          - e7c2fcae07cbada050a0af3357491d7b # PCM
          - 8e109e50e0a0b83a5098b056e13bf6db # DTS-HD HRA
          - 185f1dd7264c4562b9022d963ac37424 # DD+
          - f9f847ac70a0af62ea4a08280b859636 # DTS-ES
          - 1c1a4c5e823891c75bc50380a6866f73 # DTS
          - 240770601cc226190c367ef59aba7463 # AAC
          - c2998bd0d90ed5621d8df281e839436e # DD
          - 6ba9033150e7896bdc9ec4b44f2b230f # MP3
          - a061e2e700f81932daf888599f8a8273 # OPUS
          - b974a6cd08c1066250f1f177d7aa1225 # HDR10+
          - dfb86d5941bc9075d6af23b09c2aeecd # HDR10
          - e61e28db95d22bedcadf030b8f156d96 # HDR
          - 2a4d9069cc1fe3242ff9bdaebed239bb # HDR (undefined)
          - 08d6d8834ad9ec87b1dc7ec8148e7a1f # PQ
          - 9364dd386c9b4a1100dde8264690add7 # HLG
          - 0f12c086e289cf966fa5948eac571f44 # Hybrid
          - 570bc9ebecd92723d2d21500f4be314c # Remaster
          - eca37840c13c6ef2dd0262b141a5482f # 4K Remaster
          - e0c07d59beb37348e975a930d5e50319 # Criterion Collection
          - e9001909a4c88013a359d0b9920d7bea # Theatrical Cut
          - eecf3a857724171f968a66cb5719e152 # IMAX
          - 9f6cbff8cfe4ebbc1bde14c7b7bec0de # IMAX Enhanced 
          - 923b6abef9b17f937fab56cfcf89e1f1 # DV (WEBDL)
          - e23edd2482476e595fb990b12e7c609c # Dolby Vision HDR10
          - 58d6a88f13e2db7f5059c41047876f00 # Dolby Vision
          - 55d53828b9d81cbe20b02efd00aa0efd # Dolby Vision HLG
          - a3e19f8f627608af0211acd02bf89735 # Dolby Vision SDR
          - e7718d7a3ce595f289bfee26adc178f5 # Repack/Proper
          - ae43b294509409a6a13919dedd4764c4 # Repack2
          - b6832f586342ef70d9c128d40c07b872 # Bad Dual Groups
          - 839bea857ed2c0a8e084f3cbdbd65ecb # x265 (no HDR/DV)
          - b124be9b146540f8e62f98fe32e49a2a # 1.0 Mono
          - 89dac1be53d5268a7e10a19d3c896826 # 2.0 Stereo
          - 205125755c411c3b8622ca3175d27b37 # 3.0 
          - 373b58bd188fc00c817bd8c7470ea285 # 4.0
          - 77ff61788dfe1097194fd8743d7b4524 # 5.1 Surround
          - 6fd7b090c3f7317502ab3b63cc7f51e3 # 6.1
          - e77382bcfeba57cb83744c9c5449b401 # 7.1 Surround
          - f2aacebe2c932337fe352fa6e42c1611 # 9.1
          - b3b3a6ac74ecbd56bcdbefa4799fb9df # AMZN
          - 3472d276482257d68f7836a55ca24877 # APTV
          - 84272245b2988854bfb76a16e60baea5 # DSNP
          - 5763d1b0ce84aff3b21038eea8e9b8ad # HMAX
          - 526d445d4c16214309f0fd2b3be18a89 # Hulu
          - 170b1d363bd8516fbf3a3eb05d4faff6 # NF
          - c9fd353f8f5f1baf56dc601c4cb29920 # PCOK
          - e36a0ba1bc902b26ee40818a1d59b8bd # PMTP
          - 2a6039655313bf5dab1e43523b62c374 # MA
          - ae9b7c9ebde1f3bd336a8cbd1ec4c5e5 # No-RlsGroup
        quality_profiles:
          - name: HD-1080p
```

**Comment by rcdailey** [comment]
Are you on version 2.3.1 or greater? If not, this is a duplicate of #107.

**Comment by 2fst4u** [comment]
Facepalm. Yea sorry it was because it wasn't up to date but I'll lay out how in case someone else does this stupid mistake.

I'm on kubernetes, so functionally docker. Kubernetes will automatically set the flag `imagePullPolicy: Always` if you set the image tag as `latest`, but if you omit latest, or set another tag initially, the image will be cached and not actually pull the latest version.

If you start on `latest` you'll never run into this issue. If you start on another tag then you need to set `imagePullPolicy: Always` for it to actually be latest.

Stupid mistake on my part but only after I noticed the version wasn't set right.

**Comment by rcdailey** [comment]
No problem! I'm glad this was easy to fix. I do my best to communicate breaking changes that I'm in control of (those specifically involving Recyclarr) but unfortunately if the guide causes some breakage, that can't be communicated very well.
